src/test_utils.rs

```diff
@@ -227,24 +227,55 @@ impl TestContextManager {
         self.exec_securejoin_qr(scanner, scanned, &qr).await
     }
 
-    /// Executes SecureJoin initiated by `scanner` scanning `qr` generated by `scanned`.
+    /// Executes SecureJoin initiated by `joiner` scanning `qr` generated by `inviter`.
     ///
     /// The [`ChatId`] of the created chat is returned, for a SetupContact QR this is the 1:1
-    /// chat with `scanned`, for a SecureJoin QR this is the group chat.
+    /// chat with `inviter`, for a SecureJoin QR this is the group chat.
     pub async fn exec_securejoin_qr(
         &self,
-        scanner: &TestContext,
-        scanned: &TestContext,
+        joiner: &TestContext,
+        inviter: &TestContext,
         qr: &str,
     ) -> ChatId {
-        let chat_id = join_securejoin(&scanner.ctx, qr).await.unwrap();
+        self.exec_securejoin_qr_multi_device(joiner, &[inviter], qr)
+            .await
+    }
+
+    /// Executes SecureJoin initiated by `joiner`
+    /// scanning `qr` generated by one of the `inviters` devices.
+    /// All of the `inviters` devices will get the messages and send replies.
+    ///
+    /// The [`ChatId`] of the created chat is returned, for a SetupContact QR this is the 1:1
+    /// chat with `inviter`, for a SecureJoin QR this is the group chat.
```

Comment:
```suggestion
    /// chat with `inviters`, for a SecureJoin QR this is the group chat.
```

Reply:
"the 1:1 chat with `inviters`"? A 1:1 chat is only with one inviter (singular).